.evergreen/.evg.yml

```diff
@@ -522,7 +522,15 @@ buildvariants:
      - name: "test"
 
 - matrix_name: "tests-jdk6-secure"
-  matrix_spec: { auth: "auth", ssl: "ssl", jdk: "jdk6", version: "*", topology: "*", os: "*" }
+  matrix_spec:  { auth: "auth", ssl: "ssl", jdk: "jdk6", version: "*", topology: "*", os: "*" }
```

Comment:
No need to include `ssl: "ssl"` then have it in the exclude spec?

Change to: `matrix_spec: { auth: "auth", ssl: "nossl", jdk: "jdk6", version: "*", topology: "*", os: "*" }`